component/src/main/java/io/siddhi/extension/io/file/FileSource.java

```diff
@@ -260,6 +262,13 @@
                         type = {DataType.BOOL},
                         defaultValue = "false"
                 ),
+                @Parameter(
+                        name = "buffer.size",
+                        description = "This parameter used to get the buffer size for mode binary.chunked.",
```

Comment:
Lets rephrase as "This parameter used to get the buffer size for binary.chunk mode.

Reply:
fixed in 834102db60dbbad24ec59094d42c8f40c39d5101